setup.py

```diff
@@ -2,13 +2,19 @@
 
 from setuptools import setup, Extension
 
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
 setup(
     name="pypredict",
-    version="1.6.2",
+    version="1.6.3",
     author="Jesse Trutna",
     author_email="jesse@spire.com",
     maintainer="Spire Global Inc",
     maintainer_email="opensource@spire.com",
+    description="Python package for satellite tracking and orbital prediction",
```

Comment:
I think we should mention that this wraps Predict explicitly:

```suggestion
    description="Interface to the Predict satellite tracking and orbital prediction library",
```

Reply:
Yup, that better, I rewrote this sentence several times and couldnt find a short description I really liked - but this is clear :slightly_smiling_face: